<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemItemsDlg</class>
 <widget class="QDialog" name="SystemItemsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Opciones de importación de módulo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/aplicacion/images/BolaAleph.png</normaloff>:/aplicacion/images/BolaAleph.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblModule">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Seleccione el módulo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbModule"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblModuleName">
        <property name="text">
         <string>Módulo que se está tratando:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Estos serán los items que se importarán. Desmarque los que no quiera que se importe</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QTableWidget" name="tableWidget">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string>Nombre</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Nueva Versión</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Nuevo Módulo</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Versión actual</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Módulo actual</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Type</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Debug</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Debug on Init</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Acción a realizar</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </column>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pbCheckAll">
            <property name="text">
             <string>Check All</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pbUnselectAll">
            <property name="text">
             <string>Uncheck All</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageDiff">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Diferencias</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QSplitter" name="splitter_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QTextEdit" name="txtDiff">
              <property name="font">
               <font>
                <family>Courier</family>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
             <widget class="QTextEdit" name="txtResult"/>
            </widget>
            <widget class="QWidget" name="layoutWidget">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">background:white</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Códigos de colores:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot;font-weight:bold; color:#ff0000; background-color:#ffa07a;&quot;&gt;Ejemplo de codigo&lt;/span&gt; Código &lt;span style=&quot; font-style:italic;&quot;&gt;eliminado&lt;/span&gt; en la nueva versión y que &lt;span style=&quot; font-style:italic;&quot;&gt;existe&lt;/span&gt; en la actual versión de sistema&lt;/p&gt;&lt;p&gt;&lt;span style=&quot;font-weight:bold; color:#000080; background-color:#00ffff;&quot;&gt;Ejemplo de codigo&lt;/span&gt; Código &lt;span style=&quot; font-style:italic;&quot;&gt;añadido&lt;/span&gt; en la nueva versión y que &lt;span style=&quot; font-style:italic;&quot;&gt;no existe&lt;/span&gt; en la actual versión de sistema&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="twDiff">
                <column>
                 <property name="text">
                  <string>Acción</string>
                 </property>
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="pbBack">
            <property name="text">
             <string>Back</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/resources.qrc">
              <normaloff>:/generales/images/previuosrecord.png</normaloff>:/generales/images/previuosrecord.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pbDiff">
     <property name="text">
      <string>Diferencias con la versión actual</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/generales/images/applyfilter.png</normaloff>:/generales/images/applyfilter.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pbCancel">
     <property name="text">
      <string>Cancelar</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/generales/images/close.png</normaloff>:/generales/images/close.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pbOk">
     <property name="text">
      <string>&amp;Ok</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/aplicacion/images/ok.png</normaloff>:/aplicacion/images/ok.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
